src/extension.ts

```diff
@@ -63,19 +65,40 @@ export function activate(context: vscode.ExtensionContext) {
   });
 
   context.subscriptions.push(issueTree);
-  context.subscriptions.push(vscode.window.registerWebviewViewProvider("trivy.helpview", helpProvider));
-  context.subscriptions.push(vscode.commands.registerCommand("trivy-vulnerability-scanner.explorer-run", () => trivyWrapper.run()));
-  context.subscriptions.push(vscode.commands.registerCommand('trivy-vulnerability-scanner.version', () => trivyWrapper.showCurrentTrivyVersion()));
-  context.subscriptions.push(vscode.commands.registerCommand('trivy-vulnerability-scanner.refresh', () => misconfigProvider.refresh()));
+  context.subscriptions.push(
+    vscode.window.registerWebviewViewProvider("trivy.helpview", helpProvider)
+  );
+  context.subscriptions.push(
+    vscode.commands.registerCommand(
+      "trivy-vulnerability-scanner.explorer-run",
+      () => trivyWrapper.run()
+    )
+  );
+  context.subscriptions.push(
+    vscode.commands.registerCommand(
+      "trivy-vulnerability-scanner.update-trivy",
+      () => trivyWrapper.updateTrivyVersion()
+    )
+  );
+  context.subscriptions.push(
+    vscode.commands.registerCommand("trivy-vulnerability-scanner.version", () =>
+      trivyWrapper.showCurrentTrivyVersion()
+    )
+  );
+  context.subscriptions.push(
+    vscode.commands.registerCommand("trivy-vulnerability-scanner.refresh", () =>
+      misconfigProvider.refresh()
+    )
+  );
 
   // The command has been defined in the package.json file
   // Now provide the implementation of the command with registerCommand
   // The commandId parameter must match the command field in package.json
-  context.subscriptions.push(vscode.commands.registerCommand(
-    "trivy-vulnerability-scanner.scan",
-    () => {
-      const trivyScanCmd = "trivy --quiet filesystem --security-checks config,vuln --exit-code=10";
-      var scanResult = runCommand(trivyScanCmd, projectRootPath.toString());
+  context.subscriptions.push(
+    vscode.commands.registerCommand("trivy-vulnerability-scanner.scan", () => {
+      const trivyScanCmd =
+        "trivy --quiet filesystem --security-checks config,vuln --exit-code=10";
+      let scanResult = runCommand(trivyScanCmd, projectRootPath.toString());
```

Comment:
Please update the commands as per the log output from the extension

```shell
2024-10-28T23:17:33-06:00	WARN	'--security-checks' is deprecated. Use '--scanners' instead.

2024-10-28T23:17:33-06:00	WARN	'--scanners config' is deprecated. Use '--scanners misconfig' instead. See https://github.com/aquasecurity/trivy/discussions/5586 for the detail.
```

Reply:
when I replaced the '--security-checks' with the '--scanners' and '--scanners config' with '--scanners misconfig' then I got this error 

<img width="335" alt="image" src="https://github.com/user-attachments/assets/9896eb9b-a65c-407c-9c0d-18275df09a40">

Reply:
Did you check the logs to see what's causing the error?

Reply:
Hi @simar7, Could you please help me with this one where I can see the logs for this error, I tried to find but was not able to find the logs in my local environment